cog-jq

```diff
@@ -1,6 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
 # Set the cog argument as the filter (default: '.')
-exec /usr/bin/jq "$COG_ARGV_0"
+RESULT=$(/usr/bin/jq "$COG_ARGV_0")
+
+# Emit JSON if the output sorta looks like JSON
+if [[ "$RESULT" == "{"* ]] || [[ "$RESULT" == "["* ]]; then
```

Comment:
When returning results from a command if you echo JSON and return something that can't be parsed as JSON (like a string for instance), the executor will crash and you wont receive a response in chat. Since `jq` allows for non-JSON responses to be returned, I figured it'd be nice to try to do the right thing by only telling Cog we're returning JSON when it looks like it.

For instance if you were to always `echo "JSON"` before every response and you ran the example in the README `@cog bundle list | jq '.status'`, Cog will crash. We've talked about having a better fallback in Cog where we print an error to chat, but we haven't implemented anything like that yet.